tests/prefetch_related/tests.py

```diff
@@ -252,7 +252,8 @@ def test_m2m_then_reverse_one_to_one_object_ids(self):
             list(Book.objects.prefetch_related('authors__authorwithage'))
 
         sql = queries[-1]['sql']
-        self.assertEqual(sql.count(str(self.author1.id)), 1, sql)
+        author_id_count = sql.count(str(self.author1.id), sql.index('WHERE'))
```

Comment:
Maybe you can make this change in a separate commit and add a test assertion something like `assertWhereContains()` and apply it to the other tests also.